app/models/space.rb

```diff
@@ -248,18 +248,14 @@ def take_snapshot
   end
 
   def max_columns
-    if graph && graph['metrics']
-      return graph['metrics'].map{|e| (e['location'] && e['location']['column'] || 0)}.max
-    else
-      return 0
-    end
+    metrics.map {|metric| metric.dig('location', 'column') || 0}.max || 0
   end
 
   def to_text
-    return "#{name}
+    return <<~TEXT#{name}
       #{description}
       #{metrics.zip(guesstimates).map {|m, g| "Metric #{m['name']}: #{g['description']}"}.join("\n")}
-    "
+    TEXT
```

Comment:
Lol I think I messed something up here. There should be a newline before `#{name}`. Right now I think it's a comment.